docs/UserGuide.md

```diff
@@ -57,6 +57,42 @@ Output:
     ____________________________________________________________
 ```
 
+### Adding recurring tasks: `addr`
+Adds multiple tasks to the list of todo tasks that occur weekly.
+
+Format: `addr <description> <optional/compulsory arguments>`
+
+List of `<optional arguments>`:
+- `p/<number>` sets the priority of the task.
+- `c/<category>` sets the category of the task.
+
+List of `<compulsory arguments>`:
+- `s/<dd-MM-yyyy>` start date of recurring tasks (inclusive).
+- `e/<dd-MM-yyyy>` end date of recurring tasks (inclusive).
+- `day/<mon/tue/wed/thu/fri/sat/sun>` day of recurring task.
+
+Example of usage:
+
+`addr tp meeting s/26-10-2020 e/27-11-2020 day/fri`
+
+`addr board games club s/26-10-2020 e/27-11-2020 day/wed p/1 c/CCA`
```

Comment:
Just a small adjustment, maybe can just change the start date to 28 instead and end on 25 as shown in your sample output.

Reply:
Rest all LGTM

Reply:
Maybe it's fine because it finds the nearest wednesday so that the user can just enter a date range without worrying about the actual starting date.

Reply:
Ahh alright i see